Assets/Scripts/Editor/ResetTransforms.cs

```diff
@@ -0,0 +1,17 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace UnityLibrary
```

Comment:
```suggestion
namespace Poached.Utility
```

Reply:
Move the file to the right directory accordingly

Reply:
It has to be inside an editor folder, so it can be poached/editor